website/content/api-docs/agent/index.mdx

```diff
@@ -246,10 +246,17 @@ In order to enable [Prometheus](https://prometheus.io/) support, you need to use
 configuration directive
 [`prometheus_retention_time`](/docs/agent/options#telemetry-prometheus_retention_time).
 
-Note: If your metric includes labels that use the same key name multiple times
-(i.e. tag=tag2 and tag=tag1), only the sorted last value (tag=tag2) will be visible on
-this endpoint due to a display issue. The complete label set is correctly applied and
-passed to external metrics providers even though it is not visible through this endpoint.
+Since Consul 1.7.2 this endpoint will also automatically switch output format if 
+the request contains an `Accept` header with a compatible MIME type such as 
+`application/openmetrics-text`. This means that recent versions of Prometheus are able to 
+scrape _without_ the `?format` parameter and will still get the correct format results 
+simplifying scrape configuration.
```

Comment:
```suggestion
`application/openmetrics-text`. Prometheus v2.5.0 and newer pass this header in scraping
queries, and so will get a compatible format by default, but the `?format` query parameter may
also be used to specify the output format manually if needed.
```

Reply:
Hmm, we actually accept several different variations of the accept header though include `text/plain` which I think means that actually all versions of Prometheus will "just work" even before they adopted the opentracing header. That's why I was intentionally vague about "recent version".

Do you think it's important enough to go and figure out specifically which versions are supported? The PR for the change conversations said "all versions" but I don't know if that was rigorously proven or just "all reasonable ones people might use".

Reply:
@banks Added a sentence that I'm hoping captures the ambiguity in older versions while preserving a specific "known good range", feel free to reword if I didn't get the behavior correct.

Reply:
Thanks Mike, I slightly tweaked the words but yeah I think that's better. If you like the tweak I'll get this merged!

Thanks!

Reply:
LGTM!